lib/actions/sessions.js

```diff
@@ -88,7 +88,7 @@ function createExitAction(type) {
         }
 
         const sessions = keys(getState().sessions.sessions);
-        if (!sessions.length) {
+        if (sessions.length === 0) {
```

Comment:
Still, I would usually write `if (!something)` instead of `if (something === false)` - even if I only wanted to compare it to `false` (not `null` etc.). If it was up to me an empty array would be falsey as well, and I'd have just written `if (!sessions)`.